src/utils.js

```diff
@@ -0,0 +1,11 @@
+export function urlFormat(url, {filename=null, version=null}={}) {
+  if (!filename || !version) {
+    throw new Error('ArgumentError: File and version are required.');
```

Comment:
Extending Error in Babel 6 caused problems IIRC, so sadly we had to stop :(

- tofumatt

On 19 January 2016 at 17:53:44, Mike Nolan (notifications@github.com) wrote:

In src/utils.js:

> @@ -0,0 +1,11 @@
> +export function urlFormat(url, {filename=null, version=null}={}) {
> +  if (!filename || !version) {
> +    throw new Error('ArgumentError: File and version are required.');
This is likely a limitation in JavaScript but can we have a more specific error type? Storing that information in the Error message and regexing for it seems like it could cause pain down the line.

—
Reply to this email directly or view it on GitHub.